src/pages/index.js

```diff
@@ -30,72 +32,219 @@ const previewText = previewImageModal.querySelector(".modal__preview-text");
 
 const cardList = document.querySelector(".cards__list");
 
+const avatarModal = document.querySelector("#avatar-modal");
+const profileAvatarButton = document.querySelector(
+  ".profile__pic-overlay_button"
+);
+
+// API
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "6e53887a-44e7-4719-8efe-74af2f636950",
+    "Content-Type": "application/json",
+  },
+});
+
+Promise.all([api.getUserInfo(), api.getInitialCards()])
+  .then(([data, initialCards]) => {
+    userInfo.setUserInfo(data);
+    userInfo.setAvatar(data.avatar);
+    section = new Section(
+      {
+        items: initialCards,
+        renderer: (item) => {
+          const cardEl = renderCard(item);
+        },
+      },
+      ".cards__list"
+    );
+    section.renderItems();
+  })
+  .catch((err) => {
+    console.error(err);
+  });
+
+let section;
+
 // Validation //
 
 const editFormElement = document.querySelector("#edit-form");
 const addFormElement = document.querySelector("#add-form");
+const avatarFormelement = document.querySelector("#avatar-form");
```

Comment:
**Could be improved**

We can get any form using their **unique** attribute `name` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms